### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by LanaDX**

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -1,25 +1,23 @@
...
-[![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
-<!-- default badges end -->
-<!-- default file list -->
-*Files to look at*:
-
-* [Default.aspx](./CS/WebSite/Default.aspx) (VB: [Default.aspx](./VB/WebSite/Default.aspx))
-* [Default.aspx.cs](./CS/WebSite/Default.aspx.cs) (VB: [Default.aspx.vb](./VB/WebSite/Default.aspx.vb))
-<!-- default file list end -->
-# How to bind ASPxGridView created at design mode to different data sources at runtime.
+# Grid View for ASP.NET Web Forms - How to bind the control created in design mode to different data sources at runtime.
```
Comment (albertov05) [inline Readme.md]:
```suggestion
# Grid View for ASP.NET Web Forms - How to bind the control created in design mode to different data sources at runtime
```

File: Readme.md [hunk 2]
```diff
@@ -1,25 +1,23 @@
...
 <!-- run online -->
 **[[Run Online]](https://codecentral.devexpress.com/e2965/)**
 <!-- run online end -->
 
+This example demonstrates how to bind [ASPxGridView](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxGridView) with autogenerated columns to different data sources at runtime. The grid is created in design mode.
+
+The project contains [ASPxRadioButtonList](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxRadioButtonList) that allows users to switch between three data sources (SqlDataSource).
```
Comment (albertov05) [inline Readme.md]:
```suggestion
The project contains an [ASPxRadioButtonList](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxRadioButtonList) control that allows users to switch between three data sources (SqlDataSource). 
```

File: Readme.md [hunk 3]
```diff
@@ -1,25 +1,23 @@
...
+## Files to Look At
+
+<!-- default file list -->
+- [Default.aspx](./CS/WebSite/Default.aspx) (VB: [Default.aspx](./VB/WebSite/Default.aspx))
+- [Default.aspx.cs](./CS/WebSite/Default.aspx.cs) (VB: [Default.aspx.vb](./VB/WebSite/Default.aspx.vb))
+<!-- default file list end -->
 
+## More Examples
 
+- [Grid View for ASP.NET Web Forms - How to Dynamically Switch the Grid's Data Source and Recreate Columns at Runtime](https://github.com/DevExpress-Examples/aspxgridview-switch-grid-data-source-and-recreate-columns-at-runtime)
```
Comment (albertov05) [inline Readme.md]: Why does this example title capitalize most words, but the other titles (see lines 1, 22, 23) do not. I think either style is fine, but can we make it consistent?

Comment (LanaDX) [reply Readme.md]:
```suggestion
- [Grid View for ASP.NET Web Forms - How to dynamically switch the Grid's data source and recreate columns at runtime](https://github.com/DevExpress-Examples/aspxgridview-switch-grid-data-source-and-recreate-columns-at-runtime)
```

**Pull request merged**